code/_onclick/other_mobs.dm

```diff
@@ -52,7 +52,8 @@
 
 			visible_message(SPAN_WARNING("[src] starts climbing onto \the [A]!"))
 			shadow.visible_message(SPAN_WARNING("[shadow] starts climbing onto \the [A]!"))
-			if(do_after(src, 50, helper))
+			var delay = 50
```

Comment:
```suggestion
			var/delay = 50
```

Reply:
Please keep codestyle solid with all other files.

Reply:
How I could have missed this detail, although that's my experience of Java leaking in my code.